KiwiLooper/scripts/controlForward/controlForward.gml

```diff
@@ -0,0 +1,9 @@
+function controlForward(targetControl, sourceControl)
```

Comment:
entire control system needs a full renaming and reorganizing. it could easily be redone with structs instead of instances, and be easier to clean up